src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -312,17 +318,17 @@ private void launchApp(String itemMenuCode) {
 
 		for (UserMenuItem u : myMenu) {
 			if (u.getCode().equals(itemMenuCode)) {
-				if ("EXIT".equalsIgnoreCase(u.getCode())) {
+				if (u.getCode().equalsIgnoreCase("EXIT")) {
 					actionExit(0);
 				} else if (u.isASubMenu()) {
-					new SubMenu(this, u.getCode(), u.getButtonLabel(), myMenu);
+					// TODO update TITLE value
+					new SubMenu(this, u.getCode(), "TODO: TITLE", myMenu);
 					break;
 				} else {
 					String app = u.getMyClass();
 					// an empty menu item
-					if ("none".equalsIgnoreCase(app)) {
+					if (app.equalsIgnoreCase("none"))
```

Comment:
new standard on develop is with values as first operand (to avoid NPE)

Reply:
restored current informatici version